### ISSUE TITLE
imu+image

### ISSUE THREAD

**Issue by SYSUHPK** [rosjava/android_apps#83]
Is there an app that can get the imu+image data?

**Comment by jubeira** [comment]
Not in this repository, but indeed it would be a great idea to do a sample app that does that in Java.

**Comment by oleg-alexandrov** [comment]
Yes, please, an app that can publish camera images would be very, very useful.
